### ISSUE TITLE
dtcw generateDeck: $HOME/docToolchain-latest/resources: No such file or directory

### ISSUE THREAD

**Issue by mh182** [docToolchain/docToolchain#1118]
With the upgrade to Asciidoctor Gradle Plugin (see commit 4556c352) we removed the obsolete scripts `resources/clone.sh` and `resources/clone.ps1` (which cloned the `reveal.js` git repository).

This breaks backwards compatibility with existing `dtcw` scripts of docToolchain 2.2.1 and before.

**To Reproduce**
1. Create test project folder and download `dtcw` for docToolchain 2.2.1
```
➜ mkdir test && cd test
➜ curl -Lo dtcw doctoolchain.github.io/dtcw
➜ chmod +x dtcw
```
2. Install latest docToolchain (commit dd8750fe) without `resources/clone.sh`, use a supported JDK (for example Java 11).
```
➜ DTC_VERSION=latest ./dtcw install
dtcw - docToolchain wrapper V0.38
docToolchain Vlatest
docToolchain as CLI available
docker available
home folder exists
force latest version with local install
Already up to date.
use local homefolder install /home/max/.doctoolchain/
Java Version 11
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/7.5.1/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build 

> Configure project :

Config file '/home/max/work/dtc/test-dtcw-2.2.1/docToolchainConfig.groovy' does not exist' 
[ant:input] 
[ant:input] do you want me to create a default one for you? (y, n)
<-------------> 0% CONFIGURING [6s]

> Task :install
docToolchain is installed

BUILD SUCCESSFUL in 11s
1 actionable task: 1 executed
```
3. The call `generateDeck` which tries to clone `reveal.js` exits with an error
```
➜ DTC_VERSION=latest ./dtcw generateDeck
dtcw - docToolchain wrapper V0.38
docToolchain Vlatest
docToolchain as CLI available
docker available
home folder exists
cloning reveal.js
./dtcw: line 133: cd: /home/max/.doctoolchain/docToolchain-latest/resources: No such file or directory
```

**Expected behavior**
`dtcw` version `0.38` and before do not break with the new docToolchain release.

**Configuration**
 - docToolchain version: commit dd8750fe
 - OS: Linux, Debian unstable
 - Relevant SDK Versions: JDK 11

**Comment by mh182** [comment]
@PacoVK Could you reintegrate the scripts again. We need those for backwards compatibility.

**Comment by PacoVK** [comment]
@mh182  since this is required for backwards compatibility, what do you think should we introduce a switch to enable these scripts? Feels a bit weird if you use a newer version that uses another revealJS source but still downloads the one specified in `dtcw`. 

Would come up with something like `$DOWNLOAD_REVEAL_JS=0` and you can set this to `1` if you are using an older version. WDYT?

**Comment by mh182** [comment]
What I wanted to support was that someone use an older `dtcw`script with a newer docToolchain. 

Providing the `clone` scripts would prevent breaking `dtcw`. `dtcw` would download reveal.js but not use it since the new docToolchain uses the new plugin.

But I think we really should go for the "if you get this error" upgrade the `dtcw` wrapper script. Maybe we decide how to resolve this issue when we decided on a major version in #1117.

**Comment by PacoVK** [comment]
Ah thanks for removing the wood of my forefront :D I agree that the possible "fix", depending on #1117  outcome would be best to have some documentation in place that explains what to do in case of such error.

**Comment by PacoVK** [comment]
As discussed we won't fix this, but instead this must be documented as a breaking change. The fix will be the upgrade of the `dtcw` which will then be backwards compatible in case someone wants to run any older docToolchain version

**Comment by mh182** [comment]
The question is where do we document this?

**Comment by PacoVK** [comment]
Suggest to add a section to the [FAQ](https://doctoolchain.org/docToolchain/v2.0.x/010_manual/50_Frequently_asked_Questions.html) and we should maybe add an upgrade section to README of the repo
